include/model_ids.h

```diff
@@ -1,575 +1,515 @@
+    MODEL_LLL_BOWSER_PIECE_7,        // lll_geo_bowser_puzzle_piece_7
+    MODEL_LLL_BOWSER_PIECE_8,        // lll_geo_bowser_puzzle_piece_8
+    MODEL_LLL_BOWSER_PIECE_9,        // lll_geo_bowser_puzzle_piece_9
+    MODEL_LLL_BOWSER_PIECE_10,        // lll_geo_bowser_puzzle_piece_10
+    MODEL_LLL_BOWSER_PIECE_11,        // lll_geo_bowser_puzzle_piece_11
+    MODEL_LLL_BOWSER_PIECE_12,        // lll_geo_bowser_puzzle_piece_12
+    MODEL_LLL_BOWSER_PIECE_13,        // lll_geo_bowser_puzzle_piece_13
+    MODEL_LLL_BOWSER_PIECE_14,        // lll_geo_bowser_puzzle_piece_14
+    MODEL_LLL_MOVING_OCTAGONAL_MESH_PLATFORM,        // lll_geo_000B08
+    MODEL_LLL_SINKING_ROCK_BLOCK,        // lll_geo_000DD0
+    MODEL_LLL_ROLLING_LOG,        // lll_geo_000DE8
+    MODEL_LLL_WOOD_BRIDGE,        // lll_geo_000B50
+    MODEL_LLL_LARGE_WOOD_BRIDGE,        // lll_geo_000B68
+    MODEL_LLL_FALLING_PLATFORM,        // lll_geo_000B80
+    MODEL_LLL_LARGE_FALLING_PLATFORM,        // lll_geo_000B98
+    MODEL_LLL_VOLCANO_FALLING_TRAP,        // lll_geo_000EA8
+
+    // ddd
+    MODEL_DDD_BOWSER_SUB_DOOR,        // ddd_geo_000478
+    MODEL_DDD_BOWSER_SUB,        // ddd_geo_0004A0
+    MODEL_DDD_POLE,        // ddd_geo_000450
+
+    // wf
+    MODEL_WF_BREAKABLE_WALL_RIGHT,        // wf_geo_000B78
+    MODEL_WF_BREAKABLE_WALL_LEFT,        // wf_geo_000B90
+    MODEL_WF_KICKABLE_BOARD,        // wf_geo_000BA8
+    MODEL_WF_TOWER_DOOR,        // wf_geo_000BE0
+    MODEL_WF_KICKABLE_BOARD_FELLED,        // wf_geo_000BC8
+
+    // castle grounds
+    MODEL_CASTLE_GROUNDS_VCUTM_GRILL,        // castle_grounds_geo_00070C
+    MODEL_CASTLE_GROUNDS_FLAG,        // castle_grounds_geo_000660
+    MODEL_CASTLE_GROUNDS_CANNON_GRILL,        // castle_grounds_geo_000724
+
+    // bowser 2
+    MODEL_BOWSER_2_TILTING_ARENA,        // bowser_2_geo_000170
+
+    // bowser 3
+    MODEL_BOWSER_3_FALLING_PLATFORM_1,        // bowser_3_geo_000290
+    MODEL_BOWSER_3_FALLING_PLATFORM_2,        // bowser_3_geo_0002A8
+    MODEL_BOWSER_3_FALLING_PLATFORM_3,        // bowser_3_geo_0002C0
+    MODEL_BOWSER_3_FALLING_PLATFORM_4,        // bowser_3_geo_0002D8
+    MODEL_BOWSER_3_FALLING_PLATFORM_5,        // bowser_3_geo_0002F0
+    MODEL_BOWSER_3_FALLING_PLATFORM_6,        // bowser_3_geo_000308
+    MODEL_BOWSER_3_FALLING_PLATFORM_7,        // bowser_3_geo_000320
+    MODEL_BOWSER_3_FALLING_PLATFORM_8,        // bowser_3_geo_000338
+    MODEL_BOWSER_3_FALLING_PLATFORM_9,        // bowser_3_geo_000350
+    MODEL_BOWSER_3_FALLING_PLATFORM_10,        // bowser_3_geo_000368
+
+    // ttm
+    MODEL_TTM_ROLLING_LOG,        // ttm_geo_000730
+    MODEL_TTM_STAR_CAGE,        // ttm_geo_000710
+    MODEL_TTM_BLUE_SMILEY,        // ttm_geo_000D14
+    MODEL_TTM_YELLOW_SMILEY,        // ttm_geo_000D4C
+    MODEL_TTM_STAR_SMILEY,        // ttm_geo_000D84
+    MODEL_TTM_MOON_SMILEY,        // ttm_geo_000DBC
+
+    // actor model IDs
+
+    // first set of actor bins (0x54-0x63)
```

Comment:
Comment no longer relevant.

Also regarding actor entries, the way this is being laid out now doesn't reuse entries, wasting memory. I don't have a good solution proposed for checking against the largest group entries though (and memory loss is probably not _huge_), so this will be a non-blocking concern.